src/Microsoft.DocAsCode.Common/Loggers/InfoCodes.cs

```diff
@@ -20,6 +20,7 @@ public static class FullBuildReason
             public const string NoAvailableGroupCache = "NoAvailableGroupCache";
             public const string ConfigChanged = "ConfigChanged";
             public const string ForceRebuild = "ForceRebuild";
+            public const string TemplateChanged = "TemplateChanged";
```

Comment:
>TemplateChanged [](start = 51, length = 15)

would this impact Real-Time Insight service?

Reply:
I think no. This just adds a code that they do not recognize.